CLUEstering/CLUEstering.py

```diff
@@ -1,352 +1,859 @@
-import pandas as pd
-import numpy as np
+"""
+Density based clustering algorithm developed at CERN.
+"""
+
+from dataclasses import dataclass
 import random as rnd
+from math import sqrt
+from math import pi
+import sys
 import time
+import types
+from typing import Union
 import matplotlib.pyplot as plt
-import CLUEsteringCPP as Algo 
+import numpy as np
+import pandas as pd
 from sklearn.datasets import make_blobs
-from math import sqrt
+from sklearn.datasets import make_circles
+import CLUEsteringCPP as Algo
+
+def normalize_data(data, mean: float, std: float):
```

Comment:
Since you are already import sklearn you can use the StandardScaler from sklearn directly